### PR TITLE
Fe track chars

### PR DESCRIPTION
**Pull request by andrewroth**
@twinge This adds a visual indication underneath the text area field of how many characters are remaining

### CODE REVIEW

File: app/views/fe/questions/fe/_text_area_field.html.erb [hunk 1]
```diff
@@ -1,6 +1,12 @@
+<% max_length = text_area_field.max_length.to_i > 0 ? text_area_field.max_length : 255 %>
 <%= text_area_tag "answers[#{text_area_field.id}]", text_area_field.response(@answer_sheet), 
       :class => 'text ' + text_area_field.validation_class(@answer_sheet),
       :id => dom_id(text_area_field),
       :rows => 6,
       :disabled => @answer_sheet.frozen?&&!@presenter.try(:reference?),
-      :readonly => @answer_sheet.frozen?&&!@presenter.try(:reference?) %>
+      :readonly => @answer_sheet.frozen?&&!@presenter.try(:reference?),
+      :maxlength => max_length %>
+<div class="small" align="right"><span class="small notice">(Please use paragraphs for long answers)</span>
```
Comment (andrewroth) [inline app/views/fe/questions/fe/_text_area_field.html.erb]: hmm with fe-trans merged now, should these text strings all be using _('...') ?

Comment (twinge) [inline app/views/fe/questions/fe/_text_area_field.html.erb]: yes

**Comment by andrewroth** [comment]
Added _('...') for the strings.  I'm thinking maybe we should check with Scott and any other stakeholders before merging this?  As it will change the visual look to have the counts.  Maybe even we should only show the counts if a config variable is set to true?

**Comment by twinge** [comment]
I don't think you need the config variables. Feel free to double check with
scott.

**Comment by andrewroth** [comment]
Scott says he likes the idea, but will need some time to determine how it will look.  I'm not sure about merging it yet then, because eventually fe will get merged into fe-bootstrap and we'll have the some problem.  I can ask him if we can just go ahead with how it looks now..?

**Comment by twinge** [comment]
Does Rich Coy need to make some css changes? Will it look bad if you merge
it in now? Not sure what's meant by "determine how it will look"

**Comment by andrewroth** [comment]
I just asked Scott again and he says he's good to just merge it now

**Comment by twinge** [comment]
:+1:

**Pull request merged**